hathor/indexes/tokens_index.py

```diff
@@ -57,10 +58,16 @@ def iter_melt_utxos(self) -> Iterator[TokenUtxoInfo]:
         raise NotImplementedError
 
 
-class TokensIndex(ABC):
+class TokensIndex(BaseIndex):
     """ Index of tokens by token uid
     """
 
+    def init_loop_step(self, tx: BaseTransaction) -> None:
+        tx_meta = tx.get_metadata()
+        if tx_meta.voided_by:
```

Comment:
Why are we skipping voided transactions?

Reply:
As far as I understand, this index isn't interested in voided transactions, even soft-voided ones.

Reply:
I guess they should. Maybe we should check with Pedro.

Reply:
I guess we should discuss it further out of this PR. The goal here is only to keep the same behavior.

Reply:
I agree. The de-facto behavior is to leave out voided txs, so that's mainly why this should be done here too. But this is certainly something that we can re-visit and discuss outside this PR.